src/controller/storagemgr/storage.go

```diff
@@ -0,0 +1,46 @@
+package storagemgr
+
+import (
+	"controller/storagemgr/storagedriver"
+	"errors"
+	"github.com/edgexfoundry/device-sdk-go"
+	"github.com/edgexfoundry/device-sdk-go/pkg/startup"
+	"log"
+	"os"
+)
+
+
+
+const (
+	dataStorageService = "datastorage"
+	edgeDir = "/var/edge-orchestration"
+	dataStorageFilePath = edgeDir + "/datastorage/configuration.toml"
```

Comment:
Do we need edgeDir and dataStorageFilePath at the moment?

Reply:
We can do away with that but I left it to maintain code consistency in all areas. Please suggest if I should remove ?

Reply:
How about commenting them out!?

Reply:
I need the dataStorageService  & dataStorageFilePath in the code. I will add edgeDir in dataStorageFilePath  directly .

Reply:
You don't touch above code if you need them. I've just wondered why they should be there.